client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -139,6 +139,45 @@ export const getTotalPriceBreakdown = ( state, orderId, siteId = getSelectedSite
 		: null;
 };
 
+export const needsCustomsForm = createSelector(
+	( state, orderId, siteId = getSelectedSiteId( state ) ) => {
+		const form = getForm( state, orderId, siteId );
+		if ( isEmpty( form ) ) {
+			return false;
+		}
+		const originStatus = form.origin;
+		const origin =
+			originStatus.isNormalized && originStatus.selectNormalized && originStatus.normalized
+				? originStatus.normalized
+				: originStatus.values;
```

Comment:
The code to use the normalized address values or something falsy is repeated several times in this file (for example below for the destination, and also [on line 225 in `getAddressErrors`](https://github.com/Automattic/wp-calypso/blob/6f572bffd2d799f6f16429946882bec283c398c7/client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js#L225). Can we move it to a separate function so that it's easier to read?

Reply:
It got a bit bigger than I expected so I opened a new PR for this. You don't need to be the one to review it, of course :) https://github.com/Automattic/wp-calypso/pull/25591